lib/board.rb

```diff
@@ -0,0 +1,65 @@
+require 'pry'
+
+class Board
+
+  attr_accessor :cells
+
+  def initialize(cells = Array.new(9, " "))
+    @cells = cells 
+  end 
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end 
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end 
+
+  def position(input)
+    # binding.pry
+      if input.to_i > 0 && input.to_i < 10
```

Comment:
You can use [`Comparable#between?`](http://ruby-doc.org/core-2.2.3/Comparable.html#method-i-between-3F) to simplify your logic. (between is inclusive)
`if input.between?(1,9)`